lib/components/PrizePoolDropdown.jsx

```diff
@@ -1,50 +1,97 @@
-import React, { useState } from 'react'
-import { CONTRACT_ADDRESSES } from 'lib/constants'
-import { DropdownInputGroup } from 'lib/components/DropdownInputGroup'
-import { useWalletNetwork } from 'lib/hooks/useWalletNetwork'
+import React, { useContext, useEffect, useState } from 'react'
+import { isValidAddress } from '@pooltogether/utilities'
+
 import { PRIZE_POOL_TYPE } from 'lib/constants'
+import { WalletContext } from 'lib/components/WalletContextProvider'
+import { SelectInputGroup } from 'lib/components/SelectInputGroup'
+import { groupedOptions, knownCustomYieldSourceAddresses } from 'lib/data/prizePoolDropdownData'
+import { useWalletNetwork } from 'lib/hooks/useWalletNetwork'
+import { fetchPrizePoolType } from 'lib/utils/fetchPrizePoolType'
+import { poolToast } from 'lib/utils/poolToast'
 
 export const PrizePoolDropdown = (props) => {
```

Comment:
Having the dot there when there's no input looks strange to me. I'd prefer it to be blank when there's no text yet.
<img width="407" alt="Screen Shot 2021-05-12 at 1 16 00 PM" src="https://user-images.githubusercontent.com/13701258/118035617-71483f00-b339-11eb-8052-4273f4aa3ac3.png">


After pasting in `0xc00e94cb662c3520282e6f5717214004a7f26888` and clicking outside of the dropdown.
<img width="411" alt="Screen Shot 2021-05-12 at 3 45 38 PM" src="https://user-images.githubusercontent.com/13701258/118035423-2dedd080-b339-11eb-8df7-61e549a7f213.png">

Could we remove some padding on mobile? It overflows for the longer pools (Rari)
<img width="472" alt="Screen Shot 2021-05-12 at 3 04 29 PM" src="https://user-images.githubusercontent.com/13701258/118035461-3940fc00-b339-11eb-8009-2a988414122f.png">

"No options" should really be "Stake" right?
<img width="1057" alt="Screen Shot 2021-05-12 at 2 49 06 PM" src="https://user-images.githubusercontent.com/13701258/118035540-507fe980-b339-11eb-9b56-d083cf360a33.png">